### PR TITLE
Rework references.

### PR DESCRIPTION
**Pull request by 1uc**
The idea is to generate the references, but not check that there are no differences. Doing so allows us to push the changes automatically to `nmodl-references`.

### CODE REVIEW

**Review by JCGoran — APPROVED** [review]
Minor nitpick: could we put the URL of the branch of `nmodl-references` we are pushing to in the CI log? Just so we don't need to enter the URL by hand. A simple `echo "Pushed to: https://github.com/BlueBrain/nmodl-references/tree/${branch}"` would do since I think GH detects URLs. Other than that, LGTM.

File: .github/workflows/nmodl-ci.yml [hunk 1]
```diff
@@ -194,6 +195,39 @@ jobs:
         run:  |
           cmake --build . --target install
 
+      - name: Update references
+        if: matrix.config.update_references == 'On'
+        shell: bash
```
Comment (matz-e) [inline .github/workflows/nmodl-ci.yml:200]:
```suggestion
        shell: pwsh
```

**Comment by 1uc** [comment]
The commit mentions this PR. Hence, it appears in the timeline.

File: .github/workflows/nmodl-ci.yml [hunk 2]
```diff
@@ -194,6 +195,39 @@ jobs:
...
+              fi
+
+              ./generate_references.sh nmodl "$d" "$OUTPUT_DIR/$d"
+          done
+
+          cd $OUTPUT_DIR
+          tar cf ../references.tar *
+
+        env:
+          INSTALL_DIR: ${{ runner.workspace }}/install
```
[the remark this answers is not in the document]
Comment (1uc) [reply .github/workflows/nmodl-ci.yml:221]: Yes, could have been :)

**Pull request merged**